pkg/transform/cluster_report_transform.go

```diff
@@ -35,20 +35,19 @@ func (e ClusterReportExtraction) Transform() ([]Output, error) {
 		ClusterReport: clusterReport,
 	}
 
-	outputs := []Output{output}
-	return outputs, nil
+	outputs = []Output{output}
+	return
 }
 
-func genClusterReport(apiResources api.Resources) (ClusterReport, error) {
-	clusterReport := ClusterReport{}
+func genClusterReport(apiResources api.Resources) (clusterReport ClusterReport, err error) {
 
```

Comment:
Maybe more empty lines in this block?

Reply:
Actually never mind my comment, the code I'm talking about is legacy (existed before this patch).